libraries/rodeos/include/b1/rodeos/rodeos.hpp

```diff
@@ -50,6 +50,7 @@ struct rodeos_db_snapshot {
    eosio::checksum256                      irreversible_id = {};
    uint32_t                                first           = 0;
    std::optional<uint32_t>                 writing_block   = {};
+   uint32_t                                force_write_stride = 200;
```

Comment:
Since this is always set, I think it should be initialized to 0.

Reply:
Changed to zero.